Makefile

```diff
@@ -0,0 +1,13 @@
+DEBUG = -g
+CC = qcc
+LD = qcc
+
+TARGET = -Vgcc_ntox86_64
+
+CFLAGS += $(DEBUG) $(TARGET) -Wall
+LDFLAGS+= $(DEBUG) $(TARGET)
+
```

Comment:
When I do make command on the terminal, it runs the code under clean. One suggestion is to move the "all" section above "clean"

Reply:
Changed